client/src/components/LandingPage.js

```diff
@@ -5,23 +5,25 @@ import demoImage from "../images/mental-health-tracker-demoImg.jpg"
 /*
     in <p>Wanna try out our App?</p> Write more about how the app works and reel new users in.
 */
+const stuff= (<div className="landigPageContent">
```

Comment:
Can you make this more descriptive?
I was getting confused by it.

Maybe like this...

```
// This content is temporally removed
// Provide reason for removal here
const temporallly_disabled = ();
```